README.md

```diff
@@ -103,6 +103,10 @@ When in SSH:
     cd /vagrant
     python3 chaos.py
 
+## Nbody simulation status
+
+![So chaotic](data/imgs/nbody.png)
```

Comment:
Just move the image to server/static/imgs and the existing server should serve it.